mods/blocks/climbing.lua

```diff
@@ -71,4 +86,11 @@ minetest.register_craft({
 		{"blocks:steel_ingot", "blocks:steel_ingot", "blocks:steel_ingot"},
 		{"blocks:steel_ingot", "", "blocks:steel_ingot"},
 	}
+})
+-- Recycle
```

Comment:
Missing recipe for chain?
Also, as above, the recycle portion makes more sense as a cooking recipe?